editor/src/core/data-tracing/data-tracing.ts

```diff
@@ -436,6 +440,87 @@ function walkUpInnerScopesUntilReachingComponent(
   )
 }
 
+const hookCallRegex = /^(use[A-Za-z]+)\(/
+
+function getPossibleHookCall(expression: JSExpression): string | null {
+  switch (expression.type) {
+    case 'JS_IDENTIFIER':
+    case 'JS_PROPERTY_ACCESS':
+    case 'JS_ELEMENT_ACCESS':
+    case 'JSX_ELEMENT':
+    case 'JSX_MAP_EXPRESSION':
+    case 'ATTRIBUTE_VALUE':
+    case 'ATTRIBUTE_NESTED_ARRAY':
+    case 'ATTRIBUTE_NESTED_OBJECT':
+      return null
+    case 'ATTRIBUTE_FUNCTION_CALL':
+      if (expression.functionName.startsWith('use')) {
```

Comment:
Is it possible to use the hook call regexp here as well?

Reply:
We could but it would be slower and I don't need the ability to capture which part of the string is the function name because it's just this value.